### PR TITLE
[FIX] account : correct button title invoice email manually

### PR DESCRIPTION
**Pull request by Mtaylorr**
Sending invoice in emails to customers now display View Journal Entry in v16.2 instead of view Invoice.
Steps to reproduce the error :
1- Install accounting
2- Go to accounting/customers/invoices
3- Create an invoice and send it by email to the customer 4- View The email on mailhog

The reason of the error was becayse we didn't provide model_descirption to the function of sending the email, so it will take the one by default whch is Journal Entry.

opw-3289880

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me

File: addons/account/wizard/account_move_send.py [hunk 1]
```diff
@@ -365,6 +366,7 @@ def _send_mails(self, moves):
                 subtype_id=subtype.id,
                 partner_ids=partners.ids,
                 attachment_ids=attachment_ids,
+                model_description=model_description
```
Comment (nle-odoo) [inline addons/account/wizard/account_move_send.py:369]: it's a small detail, but usually we end with a , character (so next time we need to add a line, we only have a one line diff
```suggestion
                model_description=model_description,
```

**Review by h4818 — COMMENTED** [review]
There is a style check for spaces before and after the "="...
otherwise LGTM

**Review by h4818 — APPROVED** [review]

**Comment by nle-odoo** [comment]
robodoo r+

**Pull request closed without merging**